support-frontend/app/actions/CustomActionBuilders.scala

```diff
@@ -32,6 +33,15 @@ class CustomActionBuilders(
     new PrivateActionBuilder(addToken, checkToken, csrfConfig, cc.parsers.defaultBodyParser, cc.executionContext)
 
   def MaybeAuthenticatedAction: ActionBuilder[OptionalAuthRequest, AnyContent] =
+    if (featureSwitches.authenticateWithOkta.isOn)
+      PrivateAction andThen userFromAuthCookiesOrAuthServerActionBuilder
+    else
+      PrivateAction andThen new AsyncAuthenticatedBuilder(
+        asyncAuthenticationService.tryAuthenticateUser,
+        cc.parsers.defaultBodyParser,
+      )
```

Comment:
minor: I don't know if it's too hard to do, but it might be nice to extract this duplicated code into a function that takes the action to taken when `featureSwitches.authenticateWithOkta` is on as a parameter.
not really necessary but it might make the code slightly easier to read

Reply:
Refactored in daa7699661e6f5cbb9af9d2835c634e3b8253990